R/init.R

```diff
@@ -5,7 +5,7 @@
     packageStartupMessage("and see 'vignette(\"littler-faq\") for some basic questions.")
     if (Sys.info()[["sysname"]] %in% c("Linux", "Darwin")) { # nocov start
         if (unname(Sys.which("r")) == "") {
-            loc <- normalizePath(file.path("bin", .Platform$r_arch, "r", package="littler"))
+            loc <- normalizePath(system.file("bin", .Platform$r_arch, "r", package="littler"))
```

Comment:
Thank you.  I often catch myself using `file.path()` when I think of concatenating, we of course meant `system.file()` here as before.